test/defi/erc20RewardPool/rules/ContributionRules.test.ts

```diff
@@ -112,7 +112,7 @@ describe('ContributionRules', function () {
       await expect(contractAsAdmin.setERC721MultiplierLimit(30)).not.to.be
         .reverted;
 
-      expect(await contract.multiplierLimitERC271()).to.be.equal(30);
+      expect(await contract.multiplierLimitERC721()).to.be.equal(30);
     });
     it('admin should be able to call setERC1155MultiplierLimit', async function () {
       const {contractAsAdmin, contract} = await ContributionRulesSetup();
```

Comment:
Add tests for contractsLimit & idsLimit

Reply:
done -> https://github.com/thesandboxgame/sandbox-smart-contracts/pull/661/commits/77dd78ec92c94f35c793b4f133ce47cc69598f3d